README.md

```diff
@@ -1747,6 +1658,107 @@ class Employee
 ```
 **[⬆ back to top](#table-of-contents)**
 
+### Avoid fluent interfaces
+A [Fluent interface](https://en.wikipedia.org/wiki/Fluent_interface) is an object
+oriented API that aims to improve the readability of the source code by using
+[Method chaining](https://en.wikipedia.org/wiki/Method_chaining).
+
+While there can be some contexts, frequently builder objects, where this
+pattern reduces the verbosity of the code (for example the [PHPUnit Mock Builder](https://phpunit.de/manual/current/en/test-doubles.html)
+or the [Doctrine Query Builder](http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/query-builder.html)),
+more often it comes at some costs:
+
+1. Breaks [Encapsulation](https://en.wikipedia.org/wiki/Encapsulation_%28object-oriented_programming%29)
+1. Breaks [Decorators](https://en.wikipedia.org/wiki/Decorator_pattern)
+1. Is harder to [mock](https://en.wikipedia.org/wiki/Mock_object) in a test suite
+1. Makes diffs of commits harder to read
```

Comment:
Well, it depends on what is considered _better_.

1-2-3 lists may be more compatible on markdown readers that don't generate lists dinamically. Do you know any?
1-1-1 lists on the other hand are easier to write and maintain, and diffs are more readable.

Reply:
I would prefer uniformity, since in this project, it is customary to write 1-2-3 *([1](https://github.com/jupeter/clean-code-php#clean-code-php), [2](https://github.com/jupeter/clean-code-php#dont-use-a-singleton-pattern), [3](https://github.com/jupeter/clean-code-php#prefer-composition-over-inheritance), [4](https://github.com/jupeter/clean-code-php#dependency-inversion-principle-dip))*

You can make PR for change this in project.

Reply:
Changed.

I will PR 1-1-1 lists soon.